evasquez26/angular4-startup
# S1-ONE-120 Sign up form

## Story / Task

[ONE-120](https://bixlabs.atlassian.net/browse/ONE-120)

## What this PR does?

Allows the user to register into the app.


### Before 

We could not register into the application

### After

User can register into the application using his email address

### How to test / reproduce

- Run the app without being logged in
- Observe that there's a button at the top right that says 'Sign up"
- Clicking the button should redirect you to the sign up form
- All fields are required, if the user submits the form without some fields it will mark the fields that are blank with error
- Filling all the fields should let you advance to the sign up success form

### TODOs
- [x] Correct implementation with the real API
- [ ] Test the confirmation process

### Related PRs

N/A

## Migrations

NO

Thanks!
